src/codegen/templates/task/_in_stream.py.mako

```diff
@@ -27,13 +27,14 @@ class InStream:
     used in conjunction with reader classes to read samples from an
     NI-DAQmx task.
     """
-    __slots__ = ('_task', '_handle', '_interpreter', '_timeout', '_waveform_attribute_mode')
+    __slots__ = ('_task', '_handle', '_interpreter', '_timeout', '_reallocation_policy', '_waveform_attribute_mode')
 
     def __init__(self, task, interpreter):
         self._task = task
         self._handle = task._handle
         self._interpreter = interpreter
         self._timeout = 10.0
+        self._reallocation_policy = ReallocationPolicy.DO_NOT_REALLOCATE
```

Comment:
Why is this a stream property? In .NET it's an optional parameter on the memory-optimized read methods, which defaults to ToGrow.